packages/react/src/components/FormattingToolbar/FormattingToolbarController.tsx

```diff
@@ -33,107 +35,61 @@ const textAlignmentToPlacement = (
 
 export const FormattingToolbarController = (props: {
   formattingToolbar?: FC<FormattingToolbarProps>;
-  floatingOptions?: Partial<UseFloatingOptions>;
+  floatingUIOptions?: UseFloatingOptions;
 }) => {
-  const divRef = useRef<HTMLDivElement>(null);
+  const show = usePluginState(FormattingToolbarExtension, {
+    selector: (state) => state.show,
+  });
+
+  const getPosition = useCallback(
+    (editor: BlockNoteEditor<any, any, any>) => ({
+      from: editor.prosemirrorState.selection.from,
+      to: editor.prosemirrorState.selection.to,
+    }),
+    [],
+  );
```

Comment:
Ah, yea you had to put this in a callback because you don't want the virtual element being re-created. Better to fix it there, so that consumers can just use an inline function & not have to worry about using useCallback